### PR TITLE
corrected header issues in runbrick.py and survey.py

### PR DESCRIPTION
**Pull request by bradlyke**
I corrected most of the issues listed in:
https://github.com/legacysurvey/legacypipe/issues/275

Issues that were skipped are listed at the end of the comment.

### CODE REVIEW

**Comment by bradlyke** [comment]
Still debugging a few things in sweeps. Don't merge until I unbreak a couple files.

**Comment by bradlyke** [comment]
I reverted the generate_sweeps script to the master branch version. The other files should be fine. I'll checkout that sweeps file on a different branch. Should be okay to merge now.

File: py/legacypipe/survey.py [hunk 1]
```diff
@@ -398,40 +398,42 @@ def get_version_header(program_name, survey_dir, release, git_version=None):
...
     #hdr.add_record(dict(name='SURVEYID', value='DECam Legacy Survey (DECaLS)',
     #hdr.add_record(dict(name='SURVEYID', value='BASS MzLS',
-    hdr.add_record(dict(name='DRVERSIO', value=release,
+    hdr.add_record(dict(name='RELEASE', value=release,
                         comment='Survey data release number'))
     hdr.add_record(dict(name='OBSTYPE', value='object',
                         comment='Observation type'))
-    hdr.add_record(dict(name='PROCTYPE', value='tile',
+    hdr.add_record(dict(name='PROCTYPE', value='brick',
                         comment='Processing type'))
```
Comment (moustakas) [inline py/legacypipe/survey.py]: Note: I'm reverting the names of these NOAO-requested header cards.

**Comment by moustakas** [comment]
OK, as long as tests pass I think this is ready to merge.  Let's address #314 and #320 separately.

**Comment by moustakas** [comment]
One more comment:  I kept the `LSDR` variable (set to `DR8`) but this is yet another thing we'll need to remember to change for DR9...

Thanks for all your work on this @bradlyke!

**Comment by dstndstn** [comment]
the travis failure is due to restructuring in unwise_psf since the dr8 tag.  I changed the travis build file to grab that release.

**Comment by moustakas** [comment]
We're still failing, @dstndstn can you take a look please?

**Comment by dstndstn** [comment]
Done -- I forgot to make the same change on circleci that I made on travis
-- use a tagged version of unwise_psf.

**Pull request merged**